gc-cli/package.json

```diff
@@ -3,11 +3,8 @@
   "version": "0.0.0",
   "scripts": {
     "lint": "eslint src/. test/.",
-    "build": "npm run lint && npm run test && npm run compile",
-    "test": "mocha --compilers js:babel/register test/**",
-    "test:debug": "node --debug _mocha",
-    "precompile": "rimraf dist",
-    "compile": "babel src --out-dir dist"
+    "test": "mocha --compilers js:babel-core/register test/**/*spec.js",
```

Comment:
I'm fairly sure that this should be `mocha --compilers js:babel-register` as I think `babel-core/register` is no longer supported, might be the cause of the build failures?